src/Api/Controllers/AccountsController.cs

```diff
@@ -213,6 +214,13 @@ public async Task PostSetPasswordAsync([FromBody]SetPasswordRequestModel model)
             var result = await _userService.SetPasswordAsync(model.ToUser(user), model.MasterPasswordHash, model.Key);
             if (result.Succeeded)
             {
+                if (!model.OrgIdentifier.IsNullOrEmpty())
```

Comment:
Don't add a `IdentityServer4` dependency so that you can use this `IsNullOrEmpty` extension.

Just use `string.IsNullOrWhitespace()`

Reply:
Removed and used suggestion within `UserService`